.changeset/gentle-rocks-rule.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/theme-language-server-common': minor
+'theme-check-vscode': minor
+---
+
+Add support for HTML Element close tag auto-insertion
```

Comment:
Might want to add more colour to this changeset so the VS Code Extension has a richer changelog entry.